index.html

```diff
@@ -1,3 +1,60 @@
-﻿
+<!doctype html>
+<html lang="en">
+<head>
+<meta name="description" content="Demo project">
+<title>Cookie Recipe</title>
+</head>
 
+<body>
+<header>
+  <h1>The Best Chocolate Chip Cookies </h1>
+  </header>
 
+<section>
+  <img src="https://raw.githubusercontent.com/Coderbabez/2019-spring-01h-recipe-page/master/img/cookies.jpg"
+</section>
+  <section>
+  <h2>Recipe by Grandma<br>Prep Time: 45 Min</h2>
+  </section>
+
+  <section>
+  <h3>INGREDIENTS</h3>
+
+<ul>
+  <li>1 1/2 cup (3 sticks) softened butter</li>
+  <li>1 cup brown sugar</li>
+  <li>1 cup granulated sugar</li>
+  <li>1 Tbl vanilla instant pudding powder</li>
+  <li>2 Tbl milk</li>
+  <li> Tbl vanilla extract</li>
+  <li>2 eggs</li>
+  <li>4 cups all purpose flour</li>
+  <li>2 tsp baking soda</li>
+  <li>1/2 tsp salt</li>
+  <li>4 cups chocolate chips</li>
+  <li>1 cup chopped walnuts or pecans(optional)</li>
+<?section>
+  </ul>
+
+<section>
+  <h4>How to make Mama’s Recipe:</h4>
+
+  <ol>
+    <li>Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy.</li>
+    <li>Stir in pudding mix milk and vanilla extract.</li>
+    <li>Beat in eggs.</li>
+    <li>Add dry ingredients and stir until combined.</li>
+    <li>Stir in chocolate chips and nuts.</li>
+    <li>Place 1 1/2 inch balls of dough 2 inches apart on an un greased cookie sheet.</li>
+    <li>Bake 8-10 minutes or until golden brown.</li>
+  </section>
```

Comment:
Before you close the section, also close the `<ol>` tag. After the last item and before the `</section>`, add `</ol>`